patches/api/0468-Add-listing-API-for-PlayerProfile.patch

```diff
@@ -0,0 +1,80 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Flo0 <flo.roma@web.de>
+Date: Sat, 6 Apr 2024 23:15:08 +0200
+Subject: [PATCH] Add listing API for PlayerProfile
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index c6cb4f17469a8f2e60dd3e28d41402851ce5fb21..8d35bb590facb83487729fe5cca9e8450bbac9d2 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2054,6 +2054,69 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     boolean listPlayer(@NotNull Player other);
+     // Paper end
+ 
++    // Paper start - Add Listing API for PlayerProfile
++    /**
++     * Lists the {@code profile} in the tablist.
++     * Burden of tracking the profile is on the caller.
++     * Using profiles of online players is an invalid operation.
++     * Use {@link #listPlayer(Player)} instead.
++     *
++     * @param profile The {@link com.destroystokyo.paper.profile.PlayerProfile} to list.
++     * @param latency The latency of the profile in milliseconds.
++     * @param gameMode The game mode of the profile.
++     * @param displayName The display name of the profile.
++     */
++    void listProfile(@NotNull com.destroystokyo.paper.profile.PlayerProfile profile, int latency, @NotNull GameMode gameMode, @Nullable net.kyori.adventure.text.Component displayName);
```

Comment:
I'm not satisfied with the name of this method. I think it should be `send` something like the other "packet-only" methods we have in the API. Something like `sendPlayerTabListProfile` or some combinations of those words.

Reply:
Yes makes sense. Ill change those.